itest/lnd_channel_backup_test.go

```diff
@@ -600,6 +660,57 @@ func testChannelBackupRestoreCommitTypes(ht *lntest.HarnessTest) {
 		if !success {
 			break
 		}
+
+		// Test scbforceclose scenario: generate force close tx
+		// from SCB, Carol is offline forever.
+		success = ht.Run("scbforceclose_no_carol/"+tc.name,
```

Comment:
nit: formatting of multi-argument function call, here and the other cases below.

Reply:
Fixed.